ideas.txt

```diff
@@ -0,0 +1,52 @@
+options = arguments in discord slash command
```

Comment:
It doesn't look like this file actually got deleted from the commit. IIRC you need to run `git rm ideas.txt`